ir-test.cxx

```diff
@@ -62,7 +64,7 @@ namespace ir {
 
 	decltype(auto) get_dir_from_env(const char* name) {
 		const char* _tmp = std::getenv(name);
-		std::string ret = _tmp ? _tmp : ".";
+		std::string ret = _tmp ? _tmp : ::this_exe_path;
```

Comment:
I think, since we have an ability to specify tests through arguments, we may completely remove the old way (passing arguments through environment variables). Then ``::this_exe_path`` is going to be useless too.

Reply:
will remove and refactor this.

Reply:
removed all the code part getting path from environment variable